APITests/utils.py

```diff
@@ -54,7 +55,10 @@ def format(self, record):
 
 DATA_FLOW_SCHEMA_URL = "https://raw.githubusercontent.com/Sage-Bionetworks/data_flow/main/inst/data_model/dataflow_component.csv"
 
-BASE_URL = "https://schematic-dev.api.sagebionetworks.org/v1"
+# BASE_URL = "https://schematic-dev.api.sagebionetworks.org/v1"
+BASE_URL = "https://localhost/v1"
+# LATENCY_TABLE_STORE = "syn58975617"
```

Comment:
This is the table I created to run this.  I had to do some work to convert the existing "Start Test Time" to a date column.